chat-plugins/thing-of-the-day.js

```diff
@@ -386,7 +435,7 @@ let commands = {
 		if (room !== handler.room) return this.errorReply(`This command can only be used in ${handler.room.title}.`);
 		if (!this.can('mute', null, room)) return false;
 
-		target = this.splitTarget(target);
+		target = this.splitTarget(target, false);
```

Comment:
???????

Reply:
It gave me a typescript error when I did specify this.